core/templates/dev/head/domain/feedback_message/ThreadMessageSummaryObjectFactorySpec.ts

```diff
@@ -0,0 +1,55 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit tests for ThreadMessageSummaryObjectFactory.
+ */
+
+import { ThreadMessageSummaryObjectFactory } from
+  'domain/feedback_message/ThreadMessageSummaryObjectFactory';
+
+describe('Thread message summary object factory', () => {
+  let factory: ThreadMessageSummaryObjectFactory;
+
+  beforeEach(() => {
+    factory = new ThreadMessageSummaryObjectFactory();
+  });
+
+  describe('createFromBackendDict', () => {
```

Comment:
Please try to use a "pattern" for the description text of describe. I think `.methodName` is ok, so remove other patterns like `methodName` or `.methodName()` and all variants.